tests/one_taskset.rs

```diff
@@ -94,70 +94,6 @@ fn test_single_taskset() {
     assert!(goose_metrics.users == config.users.unwrap());
 }
 
-#[test]
-// Load test with a single task set containing two weighted tasks. Validate
-// that setting the host in the load test is properly recognized, and doesn't
-// otherwise affect the test.
-fn test_single_taskset_empty_config_host() {
```

Comment:
@LionsAd This got moved into the new `defaults` test, as that's what it's testing.